bsip-00XX Dynamic market fees.md

```diff
@@ -0,0 +1,183 @@
+    BSIP: TBD
+    Title: Dynamic Market Fees
+    Authors: OpenLedgerApp <https://github.com/OpenLedgerApp>
+    Status: Draft
+    Type: Protocol
+    Created: 2018-12-17
+    Updated: 2018-12-17
+    Discussion: 
+    Worker: TBD
+    
+# Abstract
+UIA owners do not have many marketing tools to increase trade volume of their assets.
+
+One of the ways to increase the trade volume can be Dynamic Market Fees that will provide additional discounts for active traders.
+For example, to set lower fees for trading larger volumes.
+
+So, instead of market fee, an asset owner can optionally set up dynamic_market_fee_percent property that may look like a following example table.
+**Please refer to the general notes below for terms (maker, taker) explanation.**
```

Comment:
Please make clear that the below is an example for **USD:X** markets (since the volume is declared in USD)

Reply:
thanks, noted